assemblies/common-resources/src/main/resources-filtered/etc/custom.properties

```diff
@@ -100,7 +100,7 @@ org.osgi.framework.system.packages.extra= \
  org.apache.commons.pool.impl, \
  org.apache.commons.vfs, \
  org.apache.commons.vfs.provider.http, \
- org.apache.commons.vfs2.*; version=2.2, \
+ org.apache.commons.vfs2.*; version=2.3, \
```

Comment:
Please use maven filtering here. Like this:
`org.apache.commons.vfs2.*; version=${commons-vfs2.version}, \`

Reply:
Done